scripts/release/version.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const fs = require('fs');
+const path = require('path');
+const readline = require('readline');
+const glob = require('glob');
+
+(async () => {
+    const newVersion = await promptVersion();
+    updatePackages(newVersion);
+})();
+
+async function promptVersion() {
+    const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+    });
+
+    try {
+        const answer = await new Promise((resolve) =>
+            rl.question('Enter a new LWC version: ', resolve)
+        );
+        return answer;
+    } catch (error) {
+        console.error(error);
+    } finally {
+        rl.close();
+    }
+}
+
+function updatePackages(newVersion) {
+    try {
+        const packagesToUpdate = getPackagesToUpdate();
+        const workspacesPackageJson = new Set(
+            packagesToUpdate.map(({ packageJson }) => packageJson.name)
+        );
+
+        for (const { packageJson } of packagesToUpdate) {
+            packageJson.version = newVersion;
+            // Look for different types of dependencies
+            // ex: dependencies, devDependencies, peerDependencies
+            const pkgDependencyTypes = Object.keys(packageJson).filter((key) =>
+                key.match(/.*[dD]ependencies.*/g)
```

Comment:
```suggestion
                key.match(/.*[dD]ependencies/)
```

The `g` global flag is not necessary here, and it should always end with `ependencies`.